pkg/datapath/loader/internal/generate-cilium-datapath-sha256/main.go

```diff
@@ -0,0 +1,46 @@
+// generate-cilium-datapath-sha256
+//
+package main
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"hash"
+	"io"
+	"os"
+)
+
+func ingestFile(h hash.Hash, filename string) error {
+	f, err := os.Open(filename)
+	if err != nil {
+		return err
+	}
+	_, err = io.Copy(h, f)
+	return err
+}
+
+func run() error {
+	h := sha256.New()
+	for _, arg := range os.Args[1:] {
+		if err := ingestFile(h, arg); err != nil {
+			return err
+		}
+	}
+	_, err := fmt.Printf(""+
+		"package loader\n"+
```

Comment:
As noted above, let's add a comment header saying:

```go
// Code generated by $(cmd). DO NOT EDIT
```

so the will be recognized by humans and tools as auto-generated. Also see https://pkg.go.dev/cmd/go#hdr-Generate_Go_files_by_processing_source

Reply:
Done.